Added two community links (#7955)
withastro/docs: src/content/docs/en/guides/deploy/microsoft-azure.mdx

## Community Resources 

- [Deploying an Astro Website to Azure Static Web Apps](https://www.blueboxes.co.uk/deploying-an-astro-website-to-azure-static-web-apps)

> - [Creating a Static Site with Astro, Azure Static Web Apps, and GitHub](https://agramont.net/blog/create-static-site-astro-azure-ssg/)
> - [Using Microsoft Entra Id EasyAuth with Astro and Azure Static Web App](https://agramont.net/blog/entra-id-easyauth-with-astro/)

Reviewer: ```suggestion
- [Deploying a Static Astro Site to Azure Static Web Apps using GitHub Actions](https://agramont.net/blog/create-static-site-astro-azure-ssg/#automate-deployment-with-github-actions)
```
It seems more helpful to link to the part about deployment specifically, I think!
